components/sysutils/sysstat/Makefile

```diff
@@ -0,0 +1,48 @@
+#
```

Comment:
Perhaps, but

1) There is no "diagnostic/" directory
2) There are many packages with "diagnostic/" FMRI, mostly sniffers and local load analyzers
3) The closest equivalent to sysstat is "diagnostic/top" which is why I picked this FMRI patern